bench/irmin-pack/trace_stat_summary_utils.ml

```diff
@@ -88,7 +88,7 @@ let create_pp_seconds examples =
       Float.neg_infinity examples
   in
   let finite_pp =
-    if absmax >= 60. then fun ppf v -> Mtime.Span.pp_float_s ppf v
+    if absmax >= 60. then fun ppf v -> Fmt.uint64_ns_span ppf (Int64.of_float v)
```

Comment:
I kinda made this bit up to be honest, not sure what you would like to do here?

Reply:
Your choice seems reasonable to me since I think it is close to what was there with the previous version of mtime.